include/constants/battle.h

```diff
@@ -219,10 +219,10 @@
 // Per-side statuses that affect an entire party
 #define SIDE_STATUS_REFLECT                 (1 << 0)
 #define SIDE_STATUS_LIGHTSCREEN             (1 << 1)
-#define SIDE_STATUS_STICKY_WEB              (1 << 2)
+#define SIDE_STATUS_STICKY_WEB              (1 << 2 | 1 << 3)
```

Comment:
What's the rational for using two status bits for this and future attacks?

Reply:
Maybe someone more knowledgeable then me can comment on why it needs 2 bits but as far as I can tell the change in this PR is only visually.

Reply:
What do you mean? Sticky Web before was just using one flag `0x4`, but you changed it to use two flags `0x4` and `0x8`. Yes, ultimately it functions identically, but there's now an extra bit that's being wasted.

Reply:
This change came from my never submitted PR for the combined moves system + pledge moves, where I made the change for readability reasons after noticing that some of the `SIDE_STATUS` flags were skipping a value.

For example, after `SIDE_STATUS_STICKY_WEB` which has a value of 0x4, what's defined is a `SIDE_STATUS_SPIKES` with a value of 0x10, effectively skipping a `SIDE_STATUS` 0x8 flag entirely.

When I locally made it so all the flags were defined one after the next *(1 << 0, 1 << 1, 1 << 2, 1 << 3, 1 << 4, 1 << 5 and so on)*, I noticed that stuff broke until I reverted it, so I just assumed that all the values that were skipped were actually being used but not defined explicitly.

Alex just copied that change as is.

Reply:
Ah, I can say for certain then those values certainly aren’t being used for Sticky Web. Using both bits will likely cause whatever issues you discovered before. You can leave a comment saying that the 0x8 bit for example is used for some unknown flag and shouldn’t be used.

Reply:
> What do you mean? Sticky Web before was just using one flag 0x4, but you changed it to use two flags 0x4 and 0x8. Yes, ultimately it functions identically, but there's now an extra bit that's being wasted.

Yes, I see. 

Since I didn't have a full understanding of it, I shouldn't have add those bits. That's my bad.